master/buildbot/monkeypatches/__init__.py

```diff
@@ -59,7 +59,12 @@ def patch_testcase_assert_raises_regexp():
 
 @onlyOnce
 def patch_mysqlclient_warnings():
-    from _mysql_exceptions import Warning
+    try:
+        from _mysql_exceptions import Warning
+        import mysqlclient
+        [mysqlclient]  # noqa
```

Comment:
Please try following:

```
import mysqlclient  # noqa pylint: disable=unused-import
```

instead of

```
import mysqlclient
[mysqlclient]  # noqa
```